### PR TITLE
PCC02

### PR DESCRIPTION
**Pull request by code-monk08**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): []
Estimated time spent (hours): []
Completed (yes/no): []
I stretched my coding skills (if yes what did you learn?): []
Other feedback (what can we improve?): []

### CODE REVIEW

File: 02/code-monk08/game.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    return random.choices(POUCH, k=NUM_LETTERS)
+
+
+def input_word(draw):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+    while True:
+        enteredword = input(f'enter the valid word from the letters you have drawn {draw}').upper()
```
Comment (pybites) [inline 02/code-monk08/game.py]: better entered_word

File: 02/code-monk08/game.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    answer = [word for word in answer if word in DICTIONARY]
+    return answer
+
+
+def _get_permutations_draw(draw):
+    """Helper for get_possible_dict_words to get all permutations of draw letters.
+    Hint: use itertools.permutations"""
+    answer = []
+    for n in range(1, NUM_LETTERS+1):
+        possible_words = ["".join(groups) for groups in list(
```
Comment (pybites) [inline 02/code-monk08/game.py]: [] should vertically align as per pep8

**Comment by pybites** [comment]
Good work

File: 02/code-monk08/game.py (hunk 1 above)
Comment (code-monk08) [reply 02/code-monk08/game.py]: okay, I will remember naming variable like this.

**Pull request merged**